### PR TITLE
Audit contents

### PR DESCRIPTION
**Pull request by zsombor-flds**
# Audit Content and URLs
- Added awesome_bot to validate the links for the resources.
- Removed the stale resources (e.g. dbtlearn.com).
- Updated the URLs when necessary.

_NB_: make sure to squash this PR.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,10 +27,9 @@ Happy contributing!
 
 Courses from where you can get started with Analytics Engineering.
 
+- [dbt in a real world scenario, A Beginner dbt tutorial](https://tipseason.com/dbt-tutorial-real-world-scenario-guide/) - A beginner tutorial to understand dbt with a real world example. 
 - [Mastering dbt: Beginner to Pro](https://www.udemy.com/course/mastering-dbt-data-build-tool-bootcamp/?referralCode=FFF494163B7B9E5E846F) - Paid Udemy course that covers theory, building a dbt project from scratch, and deploying to dbt Cloud.
-- [dbt in a real world scenario, A Beginner dbt tutorial](https://tipseason.com/dbt-tutorial-real-world-scenario-guide) - A beginner tutorial to understand dbt with a real world example.
 - [Analytics Engineering Glossary](https://docs.getdbt.com/glossary) - Living collection of terms & concepts commonly used in the data industry by dbt Labs.
-- [Zero to Hero dbt](https://dbtlearn.com/) - Complete course covering both theory & practice through real-world Airbnb use-case.
```
Comment (nszoni) [inline README.md:33]: working link here: https://www.udemy.com/course/complete-dbt-data-build-tool-bootcamp-zero-to-hero-learn-dbt/

**Review by nszoni — CHANGES_REQUESTED** [review]
one link needs to be updated rather than removed

File: README.md (hunk 1 above)
Comment (zsombor-flds) [reply README.md:33]: Good catch, thanks.

**Review by nszoni — APPROVED** [review]
LGTM

**Pull request merged**